tests/time_tests/scripts/run_timetest.py

```diff
@@ -90,16 +106,21 @@ def run_timetest(args: dict, log=None):
                       "Statistics aggregation is skipped.".format(args["executable"], retcode, msg))
             return retcode, {}
 
-        # Read raw statistics
+        # Read statistics from yml
```

Comment:
previous comment is more suitable because YAML reading is self-documented, and in the same time there is no info what YAML should contain